### PR TITLE
Bump ard dataset 3.2.0

### PR DESCRIPTION
**Pull request by sixy6e**
Bumping the ARD dataset version number from 3.1.0 to 3.2.0

### CODE REVIEW

File: tests/integration/test_tostac.py [hunk 1]
```diff
@@ -11,10 +11,10 @@
 from tests.integration.common import run_prepare_cli
 
 TO_STAC_DATA: Path = Path(__file__).parent.joinpath("data/tostac")
-ODC_METADATA_FILE: str = "ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml"
+ODC_METADATA_FILE: str = "ga_ls8c_ard_3-2-0_088080_2020-05-25_final.odc-metadata.yaml"
 STAC_TEMPLATE_FILE: str = "ga_ls_ard_3_stac_item.json"
 STAC_EXPECTED_FILE: str = (
-    "ga_ls8c_ard_3-1-0_088080_2020-05-25_final.stac-item_expected.json"
+    "ga_ls8c_ard_3-2-0_088080_2020-05-25_final.stac-item_expected.json"
```
Comment (jeremyh) [inline tests/integration/test_tostac.py:17]: This refers to the test data files inside the repository, so the change to them is breaking Travis. We could rename the test data too, but I'd rather revert these lines as it's the correct version the test data was made from.

File: tests/integration/data/tostac/ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 $schema: https://schemas.opendatacube.org/dataset
 id: 2aa69fcf-aa55-4747-9d95-3652f9fe79b0
 
-label: ga_ls8c_ard_3-1-0_088080_2020-05-25_final
+label: ga_ls8c_ard_3-2-0_088080_2020-05-25_final
```
Comment (jeremyh) [inline tests/integration/data/tostac/ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml:6]: Same with these two files in the `tostac` folder. Can leave them as-is. 

They're previously-packaged files, not data running through the new packager.

Comment (jeremyh) [reply tests/integration/data/tostac/ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml:6]: And if we leave them as-is now, they wont show up on future find-replaces of similar version bumps :slightly_smiling_face:

**Comment by jeremyh** [comment]
It still makes me slightly uneasy to hard-code the version like this, as anything could accidentally use the wrong eodatasets version against their input data.  

Could we maybe read the wagl version to know the dataset version, or something? (Or if preferred, throw error if wagl version isn't what's expected.)

File: tests/integration/test_tostac.py (hunk 1 above)
Comment (sixy6e) [reply tests/integration/test_tostac.py:17]: no problem. I'll revert the commits.

**Pull request closed without merging**